.github/workflows/workflow.yml

```
@@ -83,29 +167,58 @@
 
-      - name: Run Next.js Server
+      - name: Start Next.js Server
         run: pnpm --filter web start:test &
 
-      - name: Run Playwright tests
+      - name: Wait for server to be ready
+        run: |
+          echo "Waiting for server to start..."
+          sleep 10
+          # Add a simple health check if needed
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)